packages/image_picker/image_picker_for_windows/lib/image_picker_for_windows.dart

```diff
@@ -0,0 +1,143 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:image_picker_platform_interface/image_picker_platform_interface.dart';
+import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
+
+/// The Windows implementation of [ImagePickerPlatform].
+///
+/// This class implements the `package:image_picker` functionality for Windows.
+class ImagePickerWindowsPlugin extends ImagePickerPlatform {
+  /// A constructor that allows tests to override the function that creates file inputs.
+  ImagePickerWindowsPlugin();
+
+  List<String> _imageFormats = ['jpg', 'jpeg', 'png', 'bmp'];
+  List<String> _videoFormats = ['mov', 'wmv', 'mkv', 'mp4'];
+
+  /// Registers this class as the default instance of [ImagePickerPlatform].
+  static void registerWith() {
+    ImagePickerPlatform.instance = ImagePickerWindowsPlugin();
+  }
+
+  /// Returns a [PickedFile] with the image that was picked.
+  ///
+  /// The `source` argument controls where the image comes from. This can
+  /// be either [ImageSource.camera] or [ImageSource.gallery].
+  ///
+  /// Note that the `maxWidth`, `maxHeight` and `imageQuality` arguments are not supported on the Windows. If any of these arguments is supplied, it'll be silently ignored by the Windows version of the plugin.
+  ///
+  /// Use `preferredCameraDevice` to specify the camera to use when the `source` is [ImageSource.camera].
+  /// The `preferredCameraDevice` is ignored when `source` is [ImageSource.gallery]. It is also ignored if the chosen camera is not supported on the device.
+  /// Defaults to [CameraDevice.rear].
+  ///
+  /// If no images were picked, the return value is null.
+  @override
+  Future<PickedFile?> pickImage({
+    required ImageSource source,
+    double? maxWidth,
+    double? maxHeight,
+    int? imageQuality,
+    CameraDevice preferredCameraDevice = CameraDevice.rear,
+  }) async {
+    final typeGroup = XTypeGroup(label: 'images', extensions: _imageFormats);
+    final file = await FileSelectorPlatform.instance
+        .openFile(acceptedTypeGroups: [typeGroup]);
```

Comment:
Prefer implementing this by calling `getImage` so we don't have two copies of the implementation.